Packs/GroupIB_ThreatIntelligence&Attribution/Integrations/GroupIB_TIA_Feed/GroupIB_TIA_Feed.py

```diff
@@ -0,0 +1,793 @@
+    },
+    "osi/vulnerability": {
+        'indicators':
+            [
+                {
+                    'main_field': 'id', "main_field_type": 'CVE',
+                    "add_fields":
+                    [
+                        'cvss_score', 'cvss_vector', 'softwareMixed',
+                        'description', 'dateModified', 'datePublished'
+                    ],
+                    "add_fields_types":
+                    [
+                        'cvss', 'gibcvssvector', 'gibsoftwaremixed',
+                        'cvedescription', 'cvemodified', 'published'
+                    ]
+                }
+            ]
+    },
+}
+
+
+class Client(BaseClient):
+    """
+    Client will implement the service API, and should not contain any Demisto logic.
+    Should only do requests and return data.
+    """
+
+    def create_update_generator(self, collection_name: str, date_from: Optional[str] = None,
+                                seq_update: Union[int, str] = None, limit: int = 200) -> Generator:
+        """
+        Creates generator of lists with feeds class objects for an update session
+        (feeds are sorted in ascending order) `collection_name` with set parameters.
+
+        `seq_update` allows you to receive all relevant feeds. Such a request uses the seq_update parameter,
+        you will receive a portion of feeds that starts with the next `seq_update` parameter for the current collection.
+        For all feeds in the Group IB Intelligence continuous numbering is carried out.
+        For example, the `seq_update` equal to 1999998 can be in the `compromised/accounts` collection,
+        and a feed with seq_update equal to 1999999 can be in the `attacks/ddos` collection.
+        If item updates (for example, if new attacks were associated with existing APT by our specialists
+        or tor node has been detected as active again), the item gets a new parameter and it automatically rises
+        in the database and "becomes relevant" again.
+
+        :param collection_name: collection to update.
+        :param date_from: start date of update session.
+        :param seq_update: identification number from which to start the session.
+        :param limit: size of portion in iteration.
+        """
+
+        while True:
+            params = {'df': date_from, 'limit': limit, 'seqUpdate': seq_update}
+            params = {key: value for key, value in params.items() if value}
+            portion = self._http_request(method="GET", url_suffix=collection_name + '/updated',
+                                         params=params, timeout=60., retries=3)
+            if portion.get("count") == 0:
+                break
+            elif portion.get("count") < limit:
+                time.sleep(1)
+            seq_update = portion.get("seqUpdate")
+            date_from = None
+            yield portion.get('items')
+
+    def create_search_generator(self, collection_name: str, date_from: str = None,
```

Comment:
Because our update session goes from down to above and search session goes in a different direction, we use it. And they work in different ways, so it will be inconvenient to do it in one function.